.evergreen/config.yml

```diff
@@ -343,20 +343,20 @@ functions:
         binary: bash
         include_expansions_in_env: ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"]
         args:
-          - ${DRIVERS_TOOLS}/.evergreen/csfle/setup_secrets.sh
+          - ${DRIVERS_TOOLS}/.evergreen/csfle/setup-secrets.sh
```

Comment:
Should this be secrets_handling/?

Reply:
No, we only call secrets-handling when there is no convenience script already (such as for enterprise auth).